### ISSUE TITLE
Dear "lllyasviel" Can you make your own "ComfyUI" repo please?

### ISSUE THREAD

**Issue by ClothingAI** [lllyasviel/stable-diffusion-webui-forge#2072]
Hello,
your mode (flux1-dev-bnb-nf4-v2.safetensors) is really amazing, nothing matches it!
When I try other ComfyUI workflows, and their flux modes, I usually end up with 24G VRAM USAGE of my card and the UI just stop working.
For instance:
![image](https://github.com/user-attachments/assets/fddb6fec-0064-4d29-a267-f27839046f45)

___________

When I try to use your mode with other workflows, I just meet so many errors:
![image](https://github.com/user-attachments/assets/728f33d7-5ea6-4346-8c35-ae25b61222d8)

___________

OK then why don't I EXCLUSIVELY use Forge Webui?
Because LOT of extenions are NOT present in webui forge. For instance "InstantID" Which is really amazing. And many other extensions.

Let's face it! There are less and less extensions on a1111/ Forge UI.

We need you "lllyasviel" to come to the ComfyUI side, and make it "GREAT" in your own way. Please.
Can't we have both your genius with the genius of all missing extensions from comfy?
Why is your genius "limited" to the limited number of extentions classic webui (such as forge and a1111) have?

Please come to ComfyUI and make it great!
ComfyReForge.

**Comment by ClothingAI** [comment]
> 李博士计划在近期更新Forge的ControlNet。我不希望他加入Comfy阵营（节点式的操作太不容易理解了，需要极强的逻辑能力），因为他的智慧和胸怀，让我这种显存不高的设计师也可以轻松使用。

是的，但 Ilyas 会让 Comfy 变得易于使用。 Forge 的问题在于它没有 COmfy 拥有的所有工具。否则你必须创建一个“转换器”将 Comfy 节点转换为 Forge 扩展？但这会很复杂。

**Comment by Arnaud3013** [comment]
Strange, i've just update comfyui and try Flux-dev-Q8_0.gguf (using extensions for that gguf), and comfyui adapt by himself offloading in RAM. Maybe you don't have enough RAM? I've inscreased virtual RAM in windows, on my ssd (nvme) and set like 60Go for swap like. Now i do not have any issues.

**Comment by ClothingAI** [comment]
Yeah I had something ill condigured now I realized to make it work. Apparently unet + t5 GGUF is even faster

**Comment by Arnaud3013** [comment]
i've download t5 gguf, didn't tried yet. How faster? because t5 is a bit worse than fp16
